pygmt/src/which.py

```diff
@@ -27,8 +34,8 @@ def which(fname, **kwargs):
 
     Parameters
     ----------
-    fname : str
-        The file name that you want to check.
+    fname : str or list
+        One or more file names of any data type (grids, tables, etc.).
     download : bool or str
         If the file is downloadable and not found, we will try to download the
         it. Use True or 'l' (default) to download to the current directory. Use
```

Comment:
I'm tempted to update the download ([-G](https://docs.generic-mapping-tools.org/6.1/gmtwhich.html#g)) docstring to include the 'a' option (added in GMT 6.1 I think). Should it be done in this PR or a separate one?

```suggestion
    download : bool or str
        If the file is downloadable and not found, we will try to download
        it. Use True or 'l' (default) to download to the current directory. Use
```

Reply:
A separate one is better.

Reply:
> A separate one is better.

I agree.